scripts/local-testnet.sh

```diff
@@ -1,23 +1,33 @@
 CURRENT_DIR=$(pwd)
-WORKING_DIR="$CURRENT_DIR"/../..
+WORKING_DIR="$CURRENT_DIR"
 TESTNET_DIR=$WORKING_DIR/testnet
 TESTNET_OUTPUT_DIR=$TESTNET_DIR/testnet-local
-SCRIPTS_DIR=elrond-go/scripts/testnet
+SCRIPTS_DIR=mx-chain-go/scripts/testnet
 VARIABLES_PATH=$SCRIPTS_DIR/variables.sh
-ENABLE_EPOCH_DIR=$TESTNET_DIR/elrond-go/cmd/node/config/enableEpochs.toml
-SYSTEM_SC_CONFIG_DIR=$TESTNET_DIR/elrond-go/cmd/node/config/systemSmartContractsConfig.toml
+OBSERVERS_PATH=$SCRIPTS_DIR/include/observers.sh
+VALIDATORS_PATH=$SCRIPTS_DIR/include/validators.sh
+ENABLE_EPOCH_DIR=$TESTNET_DIR/mx-chain-go/cmd/node/config/enableEpochs.toml
+SYSTEM_SC_CONFIG_DIR=$TESTNET_DIR/mx-chain-go/cmd/node/config/systemSmartContractsConfig.toml
+CONFIG_TOML_DIR=$TESTNET_DIR/mx-chain-go/cmd/node/config/config.toml
 SANDBOX_NAME=sandbox
 
 cloneDependencies(){
   if [ -d "$TESTNET_DIR" ]; then
-    return
+    rm -rf $TESTNET_DIR
   fi
 
   mkdir "$TESTNET_DIR"
 
-  git clone https://github.com/ElrondNetwork/elrond-go "$TESTNET_DIR/elrond-go"
-  git clone https://github.com/ElrondNetwork/elrond-deploy-go "$TESTNET_DIR/elrond-deploy-go"
-  git clone https://github.com/ElrondNetwork/elrond-proxy-go "$TESTNET_DIR/elrond-proxy-go"
+  git clone https://github.com/multiversx/mx-chain-go "$TESTNET_DIR/mx-chain-go"
+  cd $TESTNET_DIR/mx-chain-go
```

Comment:
I would leave it as it is.
1. There are bugs/broken APIs in master and tests would fail
2. I've already spent quite some time finding why the current tests fail with newer changes from mx-chain-go and api (e.g.: there were some changes not backwards compatible for APIs). If anything new changes again in master and not backwards comp, tests would fail again. So better to use this version as a "pretty stable one"

Reply:
right, I forgot that tests uses theses scripts